extension/src/rootProject/RootProject.ts

```diff
@@ -1,12 +1,22 @@
 import * as vscode from "vscode";
+import { sendInfo } from "vscode-extension-telemetry-wrapper";
 import { Environment } from "../proto/gradle_pb";
+import { isTest } from "../util";
 
 export class RootProject {
     private environment?: Environment;
+    private gradleVersion?: string;
     constructor(private readonly workspaceFolder: vscode.WorkspaceFolder, private readonly projectUri: vscode.Uri) {}
 
     public setEnvironment(environment: Environment): void {
         this.environment = environment;
+        const gradleVersion = environment.getGradleEnvironment()?.getGradleVersion();
+        if (gradleVersion && gradleVersion !== this.gradleVersion) {
+            this.gradleVersion = gradleVersion;
+            if (!isTest()) {
+                sendInfo("", { gradleVersion: gradleVersion });
```

Comment:
consider adding something like `name: "changeGradleVersion"` to differentiate it with other info events? In other extensions, the `name` field has been tagged as identifier of `info` events.